Normalize.lua

```diff
@@ -103,3 +128,18 @@ function Normalize:__tostring__()
   end
   return string.format(s,torch.type(self),self.p)
 end
+
+function Normalize:type(type)
```

Comment:
I thought about it, but as this module doesn't have learnable parameters I thought it wasn't strictly necessary. I'll fix it (and I'll also fix the same issue with nn.Max and nn.Min)